server.R

```diff
@@ -55,4 +55,9 @@ server <- function(input, output, session) {
   subject_standards_server(id = "sas")
   learner_characteristics_server(id = "learner_characteristics")
   nps_server(id = "nps")
+  # Stop app when tab closes ==================================================
```

Comment:
I think all of these new lines (58-62) crept in during the merge conflicts, these should be deleted as we don't need them